lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -2979,7 +2979,7 @@ sub defineNewTheorem {
   my ($stomach, $flag, $thmset, $otherthmset, $type, $within) = @_;
   $thmset = ToString($thmset);
   my $classname = CleanClassName($thmset);
-  my $listname  = "theorem:$thmset"; $listname =~ s/\s//g;
+  my $listname  = "theorem:$thmset"; $listname =~ s/\s//g; $listname =~ s/'/prime/g;
```

Comment:
Should work, but kinda adhoc. I wonder if one of the existing Cleaners (eg. `CleanClassName`) might be good enough; though it appears to just strip out non-alphanumerics